[SIEM][Detections Engine] - Exceptions viewer cleanup

yctercero opened the pull request:
## Summary

This PR is a follow up to https://github.com/elastic/kibana/pull/68027 where some feedback didn't make it in. It cleans up the `and_or_badge` component, updates some css, and cleans up stories.

![Screen Shot 2020-06-09 at 9 48 46 AM](https://user-images.githubusercontent.com/10927944/84156279-4ffb1680-aa37-11ea-9aa2-e8baf7559304.png)

### Checklist
- [x] Any text added follows EUI's writing guidelines, uses sentence case text and includes i18n support
- [x] [Unit or functional tests](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility) were updated or added to match the most common scenarios
- [ ] This was checked for [keyboard-only and screenreader accessibility](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility#Accessibility_testing_checklist)
- [x] This renders correctly on smaller devices using a responsive layout. (You can test this [in your browser](https://www.browserstack.com/guide/responsive-testing-on-local-server)
- [ ] This was checked for cross-browser compatibility, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)

elasticmachine commented:
Pinging @elastic/siem (Team:SIEM)

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/)
* Commit: c74f9961cae615858ea6ac809370ea4d57477da2
* [Pipeline Steps](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/flowGraphTable) (look for red circles / failed steps)
* [Interpreting CI Failures](https://www.elastic.co/guide/en/kibana/current/interpreting-ci-failures.html)

### Failed CI Steps
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/execution/node/518/log)
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/execution/node/701/log)

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/testReport)

<details><summary>Kibana Pipeline / kibana-xpack-agent / X-Pack API Integration Tests.x-pack/test/api_integration/apis/endpoint/policy·ts.apis Endpoint plugin Endpoint policy api GET /api/endpoint/policy_response "before all" hook for "should return one policy response for host"</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/testReport//junit/X-Pack%20API%20Integration%20Tests/x-pack_test_api_integration_apis_endpoint_policy·ts/Kibana_Pipeline___kibana_xpack_agent___apis_Endpoint_plugin_Endpoint_policy_api_GET__api_endpoint_policy_response__before_all__hook_for__should_return_one_policy_response_for_host_)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 1 times on tracked branches: https://dryrun

[00:00:00]       │
[00:00:00]         └-: apis
[00:00:00]           └-> "before all" hook
[00:09:10]           └-: Endpoint plugin
[00:09:10]             └-> "before all" hook
[00:09:10]             └-> "before all" hook
[00:09:10]               │ info [o.e.x.i.IndexLifecycleTransition] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] moving index [.kibana-event-log-8.0.0-000001] from [{"phase":"hot","action":"unfollow","name":"pause-follower-index"}] to [{"phase":"hot","action":"unfollow","name":"close-follower-index"}] in policy [kibana-event-log-policy]
[00:09:10]               │ info [o.e.x.i.IndexLifecycleTransition] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] moving index [ilm-history-2-000001] from [{"phase":"hot","action":"unfollow","name":"close-follower-index"}] to [{"phase":"hot","action":"unfollow","name":"unfollow-follower-index"}] in policy [ilm-history-ilm-policy]
[00:09:10]               │ info [o.e.x.i.IndexLifecycleTransition] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] moving index [.kibana-event-log-8.0.0-000001] from [{"phase":"hot","action":"unfollow","name":"close-follower-index"}] to [{"phase":"hot","action":"unfollow","name":"unfollow-follower-index"}] in policy [kibana-event-log-policy]
[00:09:10]               │ info [o.e.x.i.IndexLifecycleTransition] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] moving index [ilm-history-2-000001] from [{"phase":"hot","action":"unfollow","name":"unfollow-follower-index"}] to [{"phase":"hot","action":"unfollow","name":"open-follower-index"}] in policy [ilm-history-ilm-policy]
[00:09:10]               │ info [o.e.x.i.IndexLifecycleTransition] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] moving index [.kibana-event-log-8.0.0-000001] from [{"phase":"hot","action":"unfollow","name":"unfollow-follower-index"}] to [{"phase":"hot","action":"unfollow","name":"open-follower-index"}] in policy [kibana-event-log-policy]
[00:09:10]               │ info [o.e.x.i.IndexLifecycleTransition] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] moving index [ilm-history-2-000001] from [{"phase":"hot","action":"unfollow","name":"open-follower-index"}] to [{"phase":"hot","action":"unfollow","name":"wait-for-yellow-step"}] in policy [ilm-history-ilm-policy]
[00:09:10]               │ info [o.e.x.i.IndexLifecycleTransition] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] moving index [.kibana-event-log-8.0.0-000001] from [{"phase":"hot","action":"unfollow","name":"open-follower-index"}] to [{"phase":"hot","action":"unfollow","name":"wait-for-yellow-step"}] in policy [kibana-event-log-policy]
[00:09:10]               │ info [o.e.x.i.IndexLifecycleTransition] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] moving index [ilm-history-2-000001] from [{"phase":"hot","action":"unfollow","name":"wait-for-yellow-step"}] to [{"phase":"hot","action":"rollover","name":"check-rollover-ready"}] in policy [ilm-history-ilm-policy]
[00:09:10]               │ info [o.e.x.i.IndexLifecycleTransition] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] moving index [.kibana-event-log-8.0.0-000001] from [{"phase":"hot","action":"unfollow","name":"wait-for-yellow-step"}] to [{"phase":"hot","action":"rollover","name":"check-rollover-ready"}] in policy [kibana-event-log-policy]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [events-settings]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [events-mappings]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [logs-mappings]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [logs-settings]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [metrics-settings]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [metrics-mappings]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [events]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [logs]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-endpoint.metadata]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-endpoint.policy]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-endpoint.telemetry]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [events-endpoint]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [logs-system.auth]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.diskio]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.core]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.filesystem]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.cpu]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.entropy]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.load]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.fsstat]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.network]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.memory]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.network_summary]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.process_summary]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.process]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.socket_summary]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.raid]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.service]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [logs-system.syslog]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.socket]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.uptime]
[00:09:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.users]
[00:09:16]             └-: Endpoint policy api
[00:09:16]               └-> "before all" hook
[00:09:16]               └-: GET /api/endpoint/policy_response
[00:09:16]                 └-> "before all" hook
[00:09:16]                 └-> "before all" hook
[00:09:16]                   │ info [endpoint/policy] Loading "mappings.json"
[00:09:16]                   │ info [endpoint/policy] Loading "data.json.gz"
[00:09:16]                   └- ✖ fail: "apis Endpoint plugin Endpoint policy api GET /api/endpoint/policy_response "before all" hook for "should return one policy response for host""
[00:09:16]                   │

```

#### Stack Trace
```
{ Error: [illegal_argument_exception] mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition
    at respond (/dev/shm/workspace/kibana/node_modules/elasticsearch/src/lib/transport.js:349:15)
    at checkRespForFailure (/dev/shm/workspace/kibana/node_modules/elasticsearch/src/lib/transport.js:306:7)
    at HttpConnector.<anonymous> (/dev/shm/workspace/kibana/node_modules/elasticsearch/src/lib/connectors/http.js:173:7)
    at IncomingMessage.wrapper (/dev/shm/workspace/kibana/node_modules/elasticsearch/node_modules/lodash/lodash.js:4929:19)
    at endReadableNT (_stream_readable.js:1145:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
  status: 400,
  displayName: 'BadRequest',
  message:
   '[illegal_argument_exception] mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition',
  path: '/metrics-endpoint.policy-default-1',
  query: {},
  body:
   { error:
      { root_cause: [Array],
        type: 'illegal_argument_exception',
        reason:
         'mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition' },
     status: 400 },
  statusCode: 400,
  response:
   '{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition"}],"type":"illegal_argument_exception","reason":"mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition"},"status":400}',
  toString: [Function],
  toJSON: [Function] }
```
</details>

---

<details><summary>Kibana Pipeline / kibana-xpack-agent / X-Pack API Integration Tests.x-pack/test/api_integration/apis/endpoint/policy·ts.apis Endpoint plugin Endpoint policy api GET /api/endpoint/policy_response "before all" hook for "should return one policy response for host"</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/testReport//junit/X-Pack%20API%20Integration%20Tests/x-pack_test_api_integration_apis_endpoint_policy·ts/Kibana_Pipeline___kibana_xpack_agent___apis_Endpoint_plugin_Endpoint_policy_api_GET__api_endpoint_policy_response__before_all__hook_for__should_return_one_policy_response_for_host__2)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches

[00:00:00]       │
[00:00:00]         └-: apis
[00:00:00]           └-> "before all" hook
[00:10:05]           └-: Endpoint plugin
[00:10:05]             └-> "before all" hook
[00:10:05]             └-> "before all" hook
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [events-settings]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [events-mappings]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [logs-mappings]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [logs-settings]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [metrics-mappings]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding component template [metrics-settings]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [events]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [logs]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-endpoint.policy]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-endpoint.metadata]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-endpoint.telemetry]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [events-endpoint]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.diskio]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.core]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [logs-system.auth]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.cpu]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.filesystem]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.entropy]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.fsstat]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.load]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.network]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.network_summary]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.memory]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.process_summary]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.raid]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.service]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.socket_summary]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [logs-system.syslog]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.process]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.socket]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.uptime]
[00:10:06]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-18-tests-xl-1591711046719756420] adding index template [metrics-system.users]
[00:10:12]             └-: Endpoint policy api
[00:10:12]               └-> "before all" hook
[00:10:12]               └-: GET /api/endpoint/policy_response
[00:10:12]                 └-> "before all" hook
[00:10:12]                 └-> "before all" hook
[00:10:12]                   │ info [endpoint/policy] Loading "mappings.json"
[00:10:12]                   │ info [endpoint/policy] Loading "data.json.gz"
[00:10:12]                   └- ✖ fail: "apis Endpoint plugin Endpoint policy api GET /api/endpoint/policy_response "before all" hook for "should return one policy response for host""
[00:10:12]                   │

```

#### Stack Trace
```
{ Error: [illegal_argument_exception] mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition
    at respond (/dev/shm/workspace/kibana/node_modules/elasticsearch/src/lib/transport.js:349:15)
    at checkRespForFailure (/dev/shm/workspace/kibana/node_modules/elasticsearch/src/lib/transport.js:306:7)
    at HttpConnector.<anonymous> (/dev/shm/workspace/kibana/node_modules/elasticsearch/src/lib/connectors/http.js:173:7)
    at IncomingMessage.wrapper (/dev/shm/workspace/kibana/node_modules/elasticsearch/node_modules/lodash/lodash.js:4929:19)
    at endReadableNT (_stream_readable.js:1145:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
  status: 400,
  displayName: 'BadRequest',
  message:
   '[illegal_argument_exception] mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition',
  path: '/metrics-endpoint.policy-default-1',
  query: {},
  body:
   { error:
      { root_cause: [Array],
        type: 'illegal_argument_exception',
        reason:
         'mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition' },
     status: 400 },
  statusCode: 400,
  response:
   '{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition"}],"type":"illegal_argument_exception","reason":"mapping fields [endpoint, @timestamp, ecs, stream, host, event] cannot be replaced during template composition"},"status":400}',
  toString: [Function],
  toJSON: [Function] }
```
</details>

---

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/","number":"52714","commit":"c74f9961cae615858ea6ac809370ea4d57477da2"}]}
PIPELINE-->

rylnd commented on x-pack/plugins/security_solution/public/common/components/and_or_badge/__examples__/index.stories.tsx:
```diff
@@ -14,26 +14,21 @@ import { AndOrBadge } from '..';
-    </ThemeProvider>
-  ))
+  .addDecorator(withTheme)
+  .add('and', () => <AndOrBadge type="and" />)
```
We don't need to do this now, but FYI: there's a storybook addon (`storybook/addon-knobs`) that allows you to define stories with interactive props. This helps to keep the navigation simpler and align closer to one story per component (instead of one story per props permutation).

yctercero replied:
Have you worked with mocking hooks in storybook. There's a bunch of different things I'd love to play around with on it. I'll take a look at knobs. Thanks!

rylnd commented on x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_details.tsx:
```diff
@@ -4,28 +4,34 @@
-    .euiDescriptionList__title.listTitle--width {
-      width: 40%;
-    }
+const MyDescriptionListTitle = styled(EuiDescriptionListTitle)`
```
Nice! Splitting them out was easier than I expected 👍

rylnd commented on x-pack/plugins/security_solution/public/common/components/exceptions/__examples__/index.stories.tsx:
```diff
@@ -27,14 +32,12 @@ storiesOf('components/exceptions', module)
+      <ExceptionItem
+        commentsAccordionId={'accordion--comments'}
+        exceptionItem={payload}
+        handleDelete={() => {}}
```
nit: My preference is to suffix these incoming callback props with `on`, and then suffix a component's internal callbacks with `handle`:

```tsx
// consumer 
<MyModal onClose={whateverConsumerWants} />

// in MyModal
const handleClose = useCallback(() => {
  handleInternalState();
  onClose();
}, [handleInternalState, onClose]);
return <EuiModal onClose={handleClose} />;
```

It's an arbitrary rule, but I find that it helps me with consistency and in distinguishing between internal/external functions.

yctercero replied:
I like that pattern. I went ahead and updated them to follow this.

rylnd commented on x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_details.tsx:
```diff
@@ -63,14 +71,14 @@ const ExceptionDetailsComponent = ({
-          />
+          <EuiDescriptionList compressed type="column" data-test-subj="exceptionsViewerItemDetails">
+            {descriptionListItems.map((item) => (
+              <Fragment key={item.title.toString()}>
```
nit: you could also use a template literal here:
```suggestion
              <Fragment key={`${item.title}`}>>
```

yctercero replied:
Thanks, updated! That looks nicer and for whatever reason feels better :)

rylnd commented on x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_entries.tsx:
```diff
@@ -134,7 +134,7 @@ const ExceptionEntriesComponent = ({
           </EuiFlexGroup>
         </EuiFlexItem>
-        <EuiFlexItem>
+        <EuiFlexItem grow={false}>
```
What caused this change, here? I couldn't see it having any effect.

yctercero replied:
It was a matter of the buttons floating to the the top of the container when the left description list was long and the number of exceptions were not. 

Here's a before:
![Screen Shot 2020-06-09 at 6 50 39 PM](https://user-images.githubusercontent.com/10927944/84209723-dd188c80-aa84-11ea-83dd-ea577f267600.png)

And after:
![Screen Shot 2020-06-09 at 9 48 46 AM](https://user-images.githubusercontent.com/10927944/84209735-e4d83100-aa84-11ea-91b2-208a6b3a49c2.png)

rylnd commented:
A few style issues I just noticed:

1. At smaller widths the RoundedBadge is broken: 
<img width="650" alt="Storybook" src="https://user-images.githubusercontent.com/657252/84181087-2f8a8680-aa4e-11ea-904f-2e5f07a1e186.png">
2. Text is wrapping for longer fields. This might be expected or wanted, but truncation might be a "cleaner" solution.
<img width="845" alt="Storybook" src="https://user-images.githubusercontent.com/657252/84181508-a293fd00-aa4e-11ea-8dc5-0203482dc67a.png">

yctercero commented on x-pack/plugins/security_solution/public/common/components/exceptions/__examples__/index.stories.tsx:
```diff
@@ -4,15 +4,20 @@
 import { getExceptionItemMock } from '../mocks';
 
+const withTheme = (storyFn: () => ReactNode) => (
+  <ThemeProvider theme={() => ({ eui: euiLightVars, darkMode: true })}>{storyFn()}</ThemeProvider>
```
I was trying to display the kibana darkmode, but it didn't actually do anything. I changed it back to false so as not to cause any confusion. I should look into that mock more.

rylnd replied:
Yeah, I think you'd need to pass euiDarkVars for that?

yctercero commented:
> A few style issues I just noticed:
> 
> 1. At smaller widths the RoundedBadge is broken:
> (See image in original comment)
> 
> 2. Text is wrapping for longer fields. This might be expected or wanted, but truncation might be a "cleaner" solution.
> (See image in original comment)

Thanks @rylnd  for pointing these out! I went ahead and 1) removed the `and` badge on smaller sizes (I can follow up with UX on this) and 2) fixed the text wrapping.

![Screen Shot 2020-06-09 at 6 48 46 PM](https://user-images.githubusercontent.com/10927944/84209872-397bac00-aa85-11ea-8e20-991c9151fd2a.png)

![Screen Shot 2020-06-09 at 6 48 37 PM](https://user-images.githubusercontent.com/10927944/84209878-3ed8f680-aa85-11ea-9244-6e4a0e8aaf1a.png)

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52904/)
* Commit: e75d12e3fe53483e4a6c0bfe5622455451a5ae4b

### History
* :broken_heart: [Build #52714](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/) failed c74f9961cae615858ea6ac809370ea4d57477da2

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/","number":"52714","commit":"c74f9961cae615858ea6ac809370ea4d57477da2"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52904/","number":"52904","commit":"e75d12e3fe53483e4a6c0bfe5622455451a5ae4b"}]}
PIPELINE-->

kibanamachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52929/)
* Commit: 5d463f94689b43b473c900a1d8a87e63bdad3746
* Flaky suites:
  * xpack-kibana-ciGroup7

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52929/testReport)

<details><summary>Kibana Pipeline / kibana-xpack-agent / X-Pack Endpoint Functional Tests.x-pack/test/functional_endpoint/apps/endpoint/policy_list·ts.endpoint When on the Endpoint Policy List and policies exists "before all" hook for "should show policy on the list"</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52929/testReport//junit/X-Pack%20Endpoint%20Functional%20Tests/x-pack_test_functional_endpoint_apps_endpoint_policy_list·ts/Kibana_Pipeline___kibana_xpack_agent___endpoint_When_on_the_Endpoint_Policy_List_and_policies_exists__before_all__hook_for__should_show_policy_on_the_list_)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 7 times on tracked branches: https://github.com/elastic/kibana/issues/66579

[00:00:00]       │
[00:00:00]         └-: endpoint
[00:00:00]           └-> "before all" hook
[00:00:00]           └-: When on the Endpoint Policy List
[00:00:00]             └-> "before all" hook
[00:00:00]             └-> "before all" hook
[00:00:00]               │ debg navigating to securitySolution url: http://localhost:6181/app/securitySolution#/management/policy
[00:00:00]               │ debg navigate to: http://localhost:6181/app/securitySolution#/management/policy
[00:00:00]               │ debg browser[INFO] http://localhost:6181/login?next=%2Fapp%2FsecuritySolution%3F_t%3D1591760605821#/management/policy 341 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:00]               │
[00:00:00]               │ debg browser[INFO] http://localhost:6181/bundles/app/core/bootstrap.js 11:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:00]               │ debg ... sleep(700) start
[00:00:00]               │ debg ... sleep(700) end
[00:00:00]               │ debg returned from get, calling refresh
[00:00:02]               │ debg browser[INFO] http://localhost:6181/login?next=%2Fapp%2FsecuritySolution%3F_t%3D1591760605821#/management/policy 341 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:02]               │
[00:00:02]               │ debg browser[INFO] http://localhost:6181/bundles/app/core/bootstrap.js 11:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:02]               │ debg currentUrl = http://localhost:6181/login?next=%2Fapp%2FsecuritySolution%3F_t%3D1591760605821#/management/policy
[00:00:02]               │          appUrl = http://localhost:6181/app/securitySolution#/management/policy
[00:00:02]               │ debg TestSubjects.find(kibanaChrome)
[00:00:02]               │ debg Find.findByCssSelector('[data-test-subj="kibanaChrome"]') with timeout=60000
[00:00:05]               │ debg browser[INFO] http://localhost:6181/33718/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js 380:106115 "INFO: 2020-06-10T03:43:30Z
[00:00:05]               │        Adding connection to http://localhost:6181/elasticsearch
[00:00:05]               │
[00:00:05]               │      "
[00:00:05]               │ debg Found login page
[00:00:05]               │ debg TestSubjects.setValue(loginUsername, test_user)
[00:00:05]               │ debg TestSubjects.click(loginUsername)
[00:00:05]               │ debg Find.clickByCssSelector('[data-test-subj="loginUsername"]') with timeout=10000
[00:00:05]               │ debg Find.findByCssSelector('[data-test-subj="loginUsername"]') with timeout=10000
[00:00:05]               │ERROR browser[SEVERE] http://localhost:6181/api/ingest_manager/check-permissions - Failed to load resource: the server responded with a status of 401 (Unauthorized)
[00:00:05]               │ debg TestSubjects.setValue(loginPassword, changeme)
[00:00:05]               │ debg TestSubjects.click(loginPassword)
[00:00:05]               │ debg Find.clickByCssSelector('[data-test-subj="loginPassword"]') with timeout=10000
[00:00:05]               │ debg Find.findByCssSelector('[data-test-subj="loginPassword"]') with timeout=10000
[00:00:05]               │ debg TestSubjects.click(loginSubmit)
[00:00:05]               │ debg Find.clickByCssSelector('[data-test-subj="loginSubmit"]') with timeout=10000
[00:00:05]               │ debg Find.findByCssSelector('[data-test-subj="loginSubmit"]') with timeout=10000
[00:00:05]               │ debg Find.findByCssSelector('[data-test-subj="kibanaChrome"] nav:not(.ng-hide)') with timeout=60000
[00:00:06]               │ debg browser[INFO] http://localhost:6181/app/securitySolution?_t=1591760605821#/management/policy 341 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:06]               │
[00:00:06]               │ debg browser[INFO] http://localhost:6181/bundles/app/core/bootstrap.js 11:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:08]               │ debg browser[INFO] http://localhost:6181/33718/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js 380:106115 "INFO: 2020-06-10T03:43:33Z
[00:00:08]               │        Adding connection to http://localhost:6181/elasticsearch
[00:00:08]               │
[00:00:08]               │      "
[00:00:08]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [metrics-index_pattern_placeholder] creating index, cause [api], templates [], shards [1]/[1], mappings [_doc]
[00:00:08]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [logs-index_pattern_placeholder] creating index, cause [api], templates [], shards [1]/[1], mappings [_doc]
[00:00:08]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [events-index_pattern_placeholder] creating index, cause [api], templates [], shards [1]/[1], mappings [_doc]
[00:00:08]               │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [.kibana_1/ITcTPC8-Q2GjdA7m48vrcA] update_mapping [_doc]
[00:00:09]               │ info [o.e.x.i.a.TransportPutLifecycleAction] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index lifecycle policy [metrics-default]
[00:00:09]               │ info [o.e.x.i.a.TransportPutLifecycleAction] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index lifecycle policy [events-default]
[00:00:09]               │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [.kibana_1/ITcTPC8-Q2GjdA7m48vrcA] update_mapping [_doc]
[00:00:09]               │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [.kibana_1/ITcTPC8-Q2GjdA7m48vrcA] update_mapping [_doc]
[00:00:09]               │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [.kibana_1/ITcTPC8-Q2GjdA7m48vrcA] update_mapping [_doc]
[00:00:09]               │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [.kibana_1/ITcTPC8-Q2GjdA7m48vrcA] update_mapping [_doc]
[00:00:09]               │ info [o.e.x.i.a.TransportPutLifecycleAction] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index lifecycle policy [logs-default]
[00:00:09]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding component template [events-mappings]
[00:00:09]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding component template [logs-mappings]
[00:00:09]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding component template [events-settings]
[00:00:09]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding component template [logs-settings]
[00:00:09]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding component template [metrics-settings]
[00:00:09]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding component template [metrics-mappings]
[00:00:09]               │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [.kibana_1/ITcTPC8-Q2GjdA7m48vrcA] update_mapping [_doc]
[00:00:09]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [events]
[00:00:09]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics]
[00:00:10]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [logs]
[00:00:10]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-endpoint.policy]
[00:00:10]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-endpoint.metadata]
[00:00:10]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-endpoint.telemetry]
[00:00:10]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [events-endpoint]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.filesystem]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.entropy]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.cpu]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.memory]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.diskio]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.core]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.network]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.fsstat]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.load]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [logs-system.auth]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.network_summary]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.service]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.raid]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.process_summary]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.socket]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.socket_summary]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [logs-system.syslog]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.uptime]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.users]
[00:00:11]               │ info [o.e.c.m.MetadataIndexTemplateService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] adding index template [metrics-system.process]
[00:00:12]               │ info [o.e.c.m.MetadataMappingService] [kibana-ci-immutable-ubuntu-16-tests-xl-1591755797252643712] [.kibana_1/ITcTPC8-Q2GjdA7m48vrcA] update_mapping [_doc]
[00:00:14]               │ debg browser[INFO] http://localhost:6181/app/securitySolution?_t=1591760620098#/management/policy 341 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution.
[00:00:14]               │
[00:00:14]               │ debg browser[INFO] http://localhost:6181/bundles/app/core/bootstrap.js 11:19 "^ A single error about an inline script not firing due to content security policy is expected!"
[00:00:14]               │ debg Finished login process currentUrl = http://localhost:6181/app/securitySolution#/management/policy
[00:00:14]               │ debg ... sleep(501) start
[00:00:15]               │ debg ... sleep(501) end
[00:00:16]               │ debg in navigateTo url = http://localhost:6181/app/securitySolution#/management/policy
[00:00:16]               │ debg TestSubjects.exists(statusPageContainer)
[00:00:16]               │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="statusPageContainer"]') with timeout=2500
[00:00:16]               │ debg browser[INFO] http://localhost:6181/33718/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js 380:106115 "INFO: 2020-06-10T03:43:41Z
[00:00:16]               │        Adding connection to http://localhost:6181/elasticsearch
[00:00:16]               │
[00:00:16]               │      "
[00:00:18]               │ debg --- retry.tryForTime error: [data-test-subj="statusPageContainer"] is not displayed
[00:00:19]               │ debg isGlobalLoadingIndicatorVisible
[00:00:19]               │ debg TestSubjects.exists(globalLoadingIndicator)
[00:00:19]               │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="globalLoadingIndicator"]') with timeout=1500
[00:00:20]               │ debg --- retry.tryForTime error: [data-test-subj="globalLoadingIndicator"] is not displayed
[00:00:21]               │ debg TestSubjects.exists(globalLoadingIndicator-hidden)
[00:00:21]               │ debg Find.existsByCssSelector('[data-test-subj="globalLoadingIndicator-hidden"]') with timeout=100000
[00:00:21]             └-> loads the Policy List Page
[00:00:21]               └-> "before each" hook: global before each
[00:00:21]               │ debg TestSubjects.exists(policyListPage)
[00:00:21]               │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="policyListPage"]') with timeout=120000
[00:00:21]               └- ✓ pass  (30ms) "endpoint When on the Endpoint Policy List loads the Policy List Page"
[00:00:21]             └-> displays page title
[00:00:21]               └-> "before each" hook: global before each
[00:00:21]               │ debg TestSubjects.getVisibleText(pageViewHeaderLeftTitle)
[00:00:21]               │ debg TestSubjects.find(pageViewHeaderLeftTitle)
[00:00:21]               │ debg Find.findByCssSelector('[data-test-subj="pageViewHeaderLeftTitle"]') with timeout=10000
[00:00:21]               └- ✓ pass  (31ms) "endpoint When on the Endpoint Policy List displays page title"
[00:00:21]             └-> shows policy count total
[00:00:21]               └-> "before each" hook: global before each
[00:00:21]               │ debg TestSubjects.getVisibleText(policyTotalCount)
[00:00:21]               │ debg TestSubjects.find(policyTotalCount)
[00:00:21]               │ debg Find.findByCssSelector('[data-test-subj="policyTotalCount"]') with timeout=10000
[00:00:21]               └- ✓ pass  (31ms) "endpoint When on the Endpoint Policy List shows policy count total"
[00:00:21]             └-> has correct table headers
[00:00:21]               └-> "before each" hook: global before each
[00:00:21]               │ debg TestSubjects.find(policyTable)
[00:00:21]               │ debg Find.findByCssSelector('[data-test-subj="policyTable"]') with timeout=10000
[00:00:21]               └- ✓ pass  (37ms) "endpoint When on the Endpoint Policy List has correct table headers"
[00:00:21]             └-> should show empty table results message
[00:00:21]               └-> "before each" hook: global before each
[00:00:21]               │ debg TestSubjects.exists(policyTable)
[00:00:21]               │ debg Find.existsByDisplayedByCssSelector('[data-test-subj="policyTable"]') with timeout=2500
[00:00:21]               │ debg TestSubjects.find(policyTable)
[00:00:21]               │ debg Find.findByCssSelector('[data-test-subj="policyTable"]') with timeout=10000
[00:00:21]               └- ✓ pass  (58ms) "endpoint When on the Endpoint Policy List should show empty table results message"
[00:00:21]             └-: and policies exists
[00:00:21]               └-> "before all" hook
[00:00:21]               └-> "before all" hook
[00:00:23]                 │ERROR {
[00:00:23]                 │        "statusCode": 500,
[00:00:23]                 │        "error": "Internal Server Error",
[00:00:23]                 │        "message": "Error connecting to package registry at https://epr.elastic.co/search?category=security: Error connecting to package registry at https://epr.elastic.co/search?category=security: Backend unavailable, connection timeout"
[00:00:23]                 │      }
[00:00:23]                 │ERROR Error: expected 200 "OK", got 500 "Internal Server Error"
[00:00:23]                 │          at Test._assertStatus (/dev/shm/workspace/kibana/node_modules/supertest/lib/test.js:268:12)
[00:00:23]                 │          at Test._assertFunction (/dev/shm/workspace/kibana/node_modules/supertest/lib/test.js:283:11)
[00:00:23]                 │          at Test.assert (/dev/shm/workspace/kibana/node_modules/supertest/lib/test.js:173:18)
[00:00:23]                 │          at assert (/dev/shm/workspace/kibana/node_modules/supertest/lib/test.js:131:12)
[00:00:23]                 │          at /dev/shm/workspace/kibana/node_modules/supertest/lib/test.js:128:5
[00:00:23]                 │          at Test.Request.callback (/dev/shm/workspace/kibana/node_modules/superagent/lib/node/index.js:718:3)
[00:00:23]                 │          at parser (/dev/shm/workspace/kibana/node_modules/superagent/lib/node/index.js:906:18)
[00:00:23]                 │          at IncomingMessage.res.on (/dev/shm/workspace/kibana/node_modules/superagent/lib/node/parsers/json.js:19:7)
[00:00:23]                 │          at IncomingMessage.emit (events.js:203:15)
[00:00:23]                 │          at endReadableNT (_stream_readable.js:1145:12)
[00:00:23]                 │          at process._tickCallback (internal/process/next_tick.js:63:19)
[00:00:23]                 │ info Taking screenshot "/dev/shm/workspace/kibana/x-pack/test/functional/screenshots/failure/endpoint When on the Endpoint Policy List and policies exists _before all_ hook.png"
[00:00:23]                 │ proc [kibana]  error  [03:43:47.934]  Error: Internal Server Error
[00:00:23]                 │ proc [kibana]     at HapiResponseAdapter.toError (/dev/shm/workspace/install/kibana-8/src/core/server/http/router/response_adapter.js:132:19)
[00:00:23]                 │ proc [kibana]     at HapiResponseAdapter.toHapiResponse (/dev/shm/workspace/install/kibana-8/src/core/server/http/router/response_adapter.js:86:19)
[00:00:23]                 │ proc [kibana]     at HapiResponseAdapter.handle (/dev/shm/workspace/install/kibana-8/src/core/server/http/router/response_adapter.js:81:17)
[00:00:23]                 │ proc [kibana]     at Router.handle (/dev/shm/workspace/install/kibana-8/src/core/server/http/router/router.js:160:34)
[00:00:23]                 │ proc [kibana]     at process._tickCallback (internal/process/next_tick.js:68:7)
[00:00:23]                 │ info Current URL is: http://localhost:6181/app/securitySolution#/management/policy?timerange=(global:(linkTo:!(timeline),timerange:(from:1591674222749,fromStr:now-24h,kind:relative,to:1591760622749,toStr:now)),timeline:(linkTo:!(global),timerange:(from:1591674222749,fromStr:now-24h,kind:relative,to:1591760622749,toStr:now)))
[00:00:23]                 │ info Saving page source to: /dev/shm/workspace/kibana/x-pack/test/functional/failure_debug/html/endpoint When on the Endpoint Policy List and policies exists _before all_ hook.html
[00:00:23]                 └- ✖ fail: "endpoint When on the Endpoint Policy List and policies exists "before all" hook for "should show policy on the list""
[00:00:23]                 │

```

#### Stack Trace
```
Error: Unable to retrieve Endpoint package via Ingest!
    at logSupertestApiErrorAndThrow (test/functional_endpoint/services/endpoint_policy.ts:57:11)
    at Object.createPolicy (test/functional_endpoint/services/endpoint_policy.ts:124:9)
```
</details>

---

### History
* :green_heart: [Build #52904](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52904/) succeeded e75d12e3fe53483e4a6c0bfe5622455451a5ae4b
* :broken_heart: [Build #52714](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/) failed c74f9961cae615858ea6ac809370ea4d57477da2

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/","number":"52714","commit":"c74f9961cae615858ea6ac809370ea4d57477da2"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52904/","number":"52904","commit":"e75d12e3fe53483e4a6c0bfe5622455451a5ae4b"},{"status":"UNSTABLE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52929/","number":"52929","commit":"5d463f94689b43b473c900a1d8a87e63bdad3746"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52943/)
* Commit: 1f006f4e5f05454f821115d9a9b0647e07ad48de

### History
* :yellow_heart: [Build #52929](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52929/) was flaky 5d463f94689b43b473c900a1d8a87e63bdad3746
* :green_heart: [Build #52904](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52904/) succeeded e75d12e3fe53483e4a6c0bfe5622455451a5ae4b
* :broken_heart: [Build #52714](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/) failed c74f9961cae615858ea6ac809370ea4d57477da2

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52714/","number":"52714","commit":"c74f9961cae615858ea6ac809370ea4d57477da2"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52904/","number":"52904","commit":"e75d12e3fe53483e4a6c0bfe5622455451a5ae4b"},{"status":"UNSTABLE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52929/","number":"52929","commit":"5d463f94689b43b473c900a1d8a87e63bdad3746"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/52943/","number":"52943","commit":"1f006f4e5f05454f821115d9a9b0647e07ad48de"}]}
PIPELINE-->

elasticmachine commented:
Pinging @elastic/security-solution (Team: SecuritySolution)
